src/fidesops/main.py

```diff
@@ -115,20 +116,27 @@ def prepare_and_log_request(
 for handler in ExceptionHandlers.get_handlers():
     app.add_exception_handler(FunctionalityNotConfigured, handler)
 
+WEBAPP_DIRECTORY = Path("src/fidesops/build/static")
+WEBAPP_INDEX = WEBAPP_DIRECTORY / "index.html"
 
-@app.on_event("startup")
-async def create_webapp_dir_if_not_exists() -> None:
```

Comment:
This function may still be useful to have. It's mostly for the case of someone is starting up the server for the first time and they've never built the frontend before, so don't have `build/static/index.html` on their local filesystem. on the fidesctl side, I added some lines to the Dockerfile to create these files, but even that doesn't work all the time since we frequently mount our own local directory over the built image.  

it's always annoying to test this, but these are the two things I double checked for while working on this on the fidesctl side:

* delete your local `build/static` directory, then navigate to localhost:8080 and make sure it doesn't error out (I _think_ without these lines it would, but am not positive, maybe it's more graceful than that!)
* build and run the docker image without volume mounting to make sure that is getting the most up to date static files properly hosted at localhost:8080

if those two are working 💯  then this is probably good to go!

Reply:
confirmed deleting `build/static` and navigating to localhost:8080 causes a 500 error

Reply:
Good find. I'll add some code that handles that situation now